### ISSUE TITLE
error: {"success":false, "message":"wrong captcha"}

### ISSUE THREAD

**Issue by bigup401** [blockchain/service-my-wallet-v3#559]
is anyone ever faced this problem on blockchain wallet service api i got this error everytime when i try to create wallet

browsers {"error":"Unexpected error, please try again"}

this shows in putty on blockchain report error: {"success":false, "message":"wrong captcha"}

**Comment by dev1ns** [comment]
Same

**Comment by bilo2bil3** [comment]
facing this problem since couple of weeks ago , can't create a single wallet

**Comment by qper228** [comment]
facing this problem since couple of weeks ago, can't create a single wallet, any response from developers ???

**Comment by dev1ns** [comment]
@qper228 Yep.. still same problem here. I contacted blockchain.com support but no help from them so far.

**Comment by dev1ns** [comment]
I also created an issue here: https://github.com/blockchain/blockchain-wallet-v4-frontend/issues/4157

**Comment by albydeca** [comment]
Getting same error

**Comment by richardsuns** [comment]
@albydeca 
@bigup401 
did you restore your blockchain wallet?  i think they are recoverable, I can recover them you can talk to me in telegrm. my telegram id is: walletrecover
